test/libsolidity/ASTJSON/assembly/empty_let_variable_declaration.json

```diff
@@ -0,0 +1,108 @@
+{
+  "absolutePath": "a",
```

Comment:
This looks a bit weird, shouldn't it be `test/libsolidity/ASTJSON/assembly/empty_let_variable_declaration.sol`?

Reply:
I think I had asked about this before, but I don't really remember it now. But, anyways, the other tests don't include the full name, just `a` or `L` (for libraries).

Reply:
Hum...right. So the [test/libsolidity/ASTJSON/assembly/empty_let_variable_declaration_parseOnly.json](https://github.com/ethereum/solidity/pull/14723/files#diff-2a812777bdf0b19c6cb7f80e1ef5b037d8836a35f61ff388ade623e22dd99779R2) should also have `a` as `absolutePath`, no?

Reply:
Yes, I corrected that already in my local branch. The only problem are different offsets in the source locations generated ... I haven't figured what the CI test does that it's different from running `solc --ast-compact-json`.

Reply:
I have fixed the tests using the output of `soltest`.